examples/pytorch/recommendation/rec/randomwalk/__init__.py

```diff
@@ -0,0 +1,61 @@
+import torch
+import dgl
+from ..utils import cuda
+from collections import Counter
+
+
+def random_walk_sampler(G, nodeset, restart_prob, max_nodes):
+    '''
+    G: DGLGraph
+    nodeset: 1D CPU Tensor of node IDs
+    restart_prob: float
+    max_nodes: int
+    return: list[list[Tensor]]
+    '''
+    traces = dgl.contrib.sampling.bipartite_single_sided_random_walk_with_restart(
+            G, nodeset, restart_prob, max_nodes)
```

Comment:
why only one random walk sampler is provided?

Reply:
This is the one we need for PinSage.  Since this is in `examples` (i.e. not a DGL interface) I only wrote one.